backend/tsconfig.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
This entire file is auto-generated, except the `outDir` config is set to be `dist/` to hold the built typescript app.